packages/next/src/app-dir/client.test.tsx

```diff
@@ -333,7 +333,7 @@ describe('with transformer', () => {
 });
 
 describe('type tests', () => {
-  const ignoreErrors = async (fn: () => Promise<unknown> | unknown) => {
```

Comment:
https://typescript-eslint.io/rules/no-redundant-type-constituents: `Promise<unknown> | unknown>` simplifies down to `unknown`. I can understand if you wanted the type to explicitly document that this might be a `Promise`. Do you have feelings on this rule?

Reply:
I have feelings but they ain't strong, feel free to nuke it